.github/workflows/sync-example-to-template.yml

```diff
@@ -0,0 +1,58 @@
+name: Sync example project to Atlas template
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'examples/next/getting-started/**'
+jobs:
+  sync_example_project_to_atlas_template:
+    runs-on: ubuntu-latest
+    env:
+      CHECKOUT_DIR_NAME: ${{ github.event.repository.name }}
+      SOURCE_REPO: ${{ github.event.repository.full_name }}
+      TARGET_REPO: wpengine/faust-atlas-hello-world
+      TARGET_BRANCH: main
+      SYNC_DIR: examples/next/getting-started # No trailing slash
+      GIT_EMAIL: bot@wpengine.com
+      GIT_NAME: WP Engine Bot 
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set SSH Agent
+        uses: webfactory/ssh-agent@v0.5.4
+        with:
+          ssh-private-key: ${{ secrets.SYNC_ATLAS_TEMPLATE_SSH_PRIVATE_KEY }}
+      - name: Sync Atlas Template
+        run: |
+          # Store the code we want to push to the target in "source" dir
+          cd ../
+          mkdir source
+
+          # Move all files in SYNC_DIR to the "source" dir
+          cd $CHECKOUT_DIR_NAME
+          mv $SYNC_DIR/* ../source/
+          cd ../
+
+          # Clone the target repo, and move it's .git dir to the "source" dir
+          git clone git@github.com:$TARGET_REPO.git target
+          cd target
+          mv .git ../source
+          cd ../
+
+          cd source
+
+          # This is the committer who will show up in the target repo
+          git config user.email "$GIT_EMAIL"
+          git config user.name "$GIT_NAME"
+
+          # Add all the recent changes non-existent in the target
+          git add .
+
+          git status
+
+          # If there are changes, commit them
+          git diff-index --quiet HEAD || git commit -m "Automatic sync from $SOURCE_REPO"
```

Comment:
Could we make the commit message reference the commit SHA in `wpengine/faustjs` that triggered the workflow? I think it could help us verify things are in sync at a glance.

Setting a var for [`github.event.after`](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push) and tacking it onto the `$SOURCE_REPO` to create the [full auto linked reference](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls#commit-shas) might work. `$SOURCE_REPO@$AFTER_SHA`

Reply:
Yeah good idea, just tried this out on a private repo, the full auto linked ref works great. Just pushed a commit for this

Reply:
Yeah that is where I am leaning on the force push side of things. Say there is an edge case, we can always manually intervene. @theodesp @apmatthews Would you be comfortable running with this implementation, and then if any issues arise we can revisit?

Reply:
I'm good with it ✅